CRM/Core/Config/Runtime.php

```diff
@@ -99,16 +92,6 @@ public function initialize($loadFromDB = TRUE) {
       $this->fatal('You need to define CIVICRM_TEMPLATE_COMPILEDIR in civicrm.settings.php');
```

Comment:
We can probably get rid of this too. `\CRM_Utils_File::baseFilePath()` already has a fatal error check.

Reply:
Good point. I'll update that.